Module3-Compliance.js

```diff
@@ -587,4 +583,4 @@ function sendExpulsionNotifications(discordHandle) {
   } else {
     Logger.log(`Error: Ambassador with discord handle: ${discordHandle} not found in the registry.`);
   }
-}
+}
```

Comment:
adjust vscode settings - your saves are adding a blank line so keep having small diffs with no functional change. Makes PRs look bigger than they actually are.

Reply:
> adjust vscode settings - your saves are adding a blank line so keep

we have ```'eol-last': ['error', 'always'],``` in eslint